### DISCUSSION TITLE
On drag, animate items around the drop indicator with the space of the dragged item

### DISCUSSION THREAD

**Discussion by thomasbritton** [adobe/react-spectrum#6871] [Q&A] [answered]
Hi,

I would like to animate the individual items in the grid so when you drag an item to a new position it appears so the cards move to the left and right to make space for the new item i have the [following code sandbox](https://codesandbox.io/p/sandbox/exciting-jones-7sysc8?file=%2Fsrc%2FApp.tsx%3A44%2C147) with what I have so far but I can't seem to get it going.

Does anyone have any ideas on how to achieve this please?

I would ideally like to use Tailwind for this if possible

**Comment by reidbarber** [comment, marked as answer]
I don't think this is possible at the moment due to collection layouts work. I think if we wanted to do this, we would essentially need to render a fake copy of the collection as if the drop had already occurred, while the user is still dragging.

Maybe there is some hack where the drop indicator could have the same styles as the dragged item(s), but I'm not sure this would work currently.

**Reply by thomasbritton** [reply]
Hey @reidbarber ,

Thanks for your reply.

I did try mimicking the dragged item with the same styles as the drop indicator but didn't make any difference.

But no issues, I was just curious
